meta-oe/recipes-devtools/perfetto/perfetto.bb

```diff
@@ -0,0 +1,151 @@
+LICENSE = "Apache-2.0 & Unknown"
```

Comment:
Which parts have unknown license?

Reply:
I checked the licenses and updated the license section accordingly:
* Perfetto: Apache-2.0
* benchmark: Apache-2.0
* googletest: BSD-3-Clause
* jsoncpp: MIT
* jsoncpp/devtools/licenseupdater.py: MIT
* libcxx: Apache-2.0
* libcxx/utils/google-benchmark: Apache-2.0
* libcxxabi: Apache-2.0
* libunwind: Apache-2.0
* linenoise: BSD-2-Clause
* protobuf: BSD-3-Clause
* sqlite: Public Domain
* zlib: ZLib
* Debian Perfetto package license: Apache-2.0
* Python module license: Apache-2.0